sass/_authorinfo.scss

```diff
@@ -0,0 +1,54 @@
+.author__info {
+  margin: 30px 10% 20px;
+  text-align: center;
+  > h1 {
+    font-size: 1.5em;
+    color: #106DB3;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+}
+.author__pic {
+  background-image: url('./images/dandysir.jpg');
+  background-size: cover;
+  background-position: 50% 50%;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100px;
+  height: 100px;
+  border-radius: 100%;
+}
+.author__picOverlay {
+  background:rgba(23, 107, 175, .5);
+  text-align: center;
+  opacity: 1;
+  width:100%;
+  min-height:100%;
+  border-radius: 100%;
+  &:hover {
+    opacity: 0;
+  }
+}
+.author__links {
+  list-style: none;
+  // margin-top: 0;
+  padding-left: 0;
+  > li {
+    display: inline-block;
+    padding: 0 10px 10px;
+    font-size: 1.1em;
+    color: #CCCCCC;
+    @media screen and (min-width: 800px) {
+      font-size: .8em;
+      padding: 0 5px 10px;
+    }
+  }
+  > li > a {
+    color: inherit;
+    text-decoration: none;
+  }
+  > li > a:hover {
```

Comment:
these 2 definitions are technically nested within the `> li`, so you could put these up after line `44`